src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -472,8 +476,15 @@ ensogl::define_endpoints! {
         visualization_enable_fullscreen (NodeId),
         visualization_set_preprocessor  ((NodeId,data::EnsoCode)),
 
+        visualization_selected          (NodeId),
+        visualization_deselected        (NodeId),
+
+        some_visualisation_selected     (bool),
+
         node_being_edited (Option<NodeId>),
-        node_editing (bool)
+        node_editing (bool),
+
+        navigation_active (bool),
```

Comment:
I'd name it `navigator_active` instead, as the namme `navigator` is used pretty often and for me, it points better at the subject of this action. Ignore if you feel otherwise.